src/flow/src/adapter/worker.rs

```diff
@@ -332,9 +346,13 @@ impl<'s> Worker<'s> {
                 let ret = self.remove_flow(flow_id);
                 Some(Response::Remove { result: ret })
             }
-            Request::RunAvail { now } => {
+            Request::RunAvail { now, blocking } => {
                 self.run_tick(now);
-                None
+                if blocking {
+                    Some(Response::RunAvail)
+                } else {
+                    None
+                }
```

Comment:
Just return `Some(Response::RunAvail)`

Reply:
> Just return `Some(Response::RunAvail)`

Had a check in here: https://github.com/GreptimeTeam/greptimedb/blob/890a2c13faa1c4e7b257157c7fa2849ca7b9ce9e/src/flow/src/adapter/worker.rs#L291
and don't want to elide it